src/config.c

```diff
@@ -1,125 +1,351 @@
 #include "config.h"
+#include "err.h"
 #include "rmutil/util.h"
 #include "rmutil/strings.h"
 #include <string.h>
 #include <stdlib.h>
+#include <limits.h>
 
-RSConfig RSGlobalConfig = RS_DEFAULT_CONFIG;
+#define RETURN_ERROR(s) return REDISMODULE_ERR;
```

Comment:
When we add richer error reporting, the string will actually be returned to the user somehow